### PR TITLE
Solution makeCalculator

### PR DESCRIPTION
**Pull request by mykola-chernysh**

### CODE REVIEW

File: src/makeCalculator.js [hunk 1]
```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: function(number) {
+      calculator.result += number;
+    },
```
Comment (stanislav-topikha) [inline src/makeCalculator.js:44]:
```suggestion
    add: function(number) {
      this.result += number;
    },
```
fix it everywhere

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]
Checklist 4-th
https://github.com/mate-academy/js_advanced_calculator/blob/master/checklist.md

File: src/makeCalculator.js (hunk 1 above)
Comment (mykola-chernysh) [reply src/makeCalculator.js:44]: > fix it everywhere

tests do not pass if I change

**Comment by stanislav-topikha** [comment]
> Checklist 4-th https://github.com/mate-academy/js_advanced_calculator/blob/master/checklist.md

Fix 4-th point

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]
⬆️

**Review by stanislav-topikha — APPROVED** [review]
GJ 👍🏻

**Pull request closed without merging**